README.md

```diff
@@ -62,18 +62,22 @@ ansible-playbook test/demo/organization.yml
 Set up debugging using ansible's test-module
 
 ```sh
-ansible-playbook debug-setup.yml
+make debug-setup
 ```
 
 Debug with ansible's test-module
 
 ```sh
-./ansible/hacking/test-module -m <path to ansible module> -a @<path to input arguments to module> -D <path to debugger>
+make debug MODULE=<module name>
 
-# Example: debug the activation_key module with pdb
-./ansible/hacking/test-module -m ./foreman-ansible-modules/modules/katello_activation_key.py -a @activation-key-args.json -D /usr/lib/python2.7/site-packages/pdb/pdb.py
+# Example: debug the katello_content_view module
+$ make debug MODULE=katello_content_view
+./.tmp/ansible/hacking/test-module -m modules/katello_content_view.py -a @test/data/content-view.json -D /usr/lib64/python2.7/pdb.py
```

Comment:
It would be nice, to have configurable server (and credentials) in these json files, too. Do you see an easy way to incorporate this?

Reply:
I agree. I can't think of anything right off, but I can look into it.

Reply:
I recall having a wrapper in another project where we override the loading and saving of the hostname, username and password to static strings for the VCR. Otherwise tests can fail when you run them with different credentials.